adapters/handlers/rest/configure_api.go

```diff
@@ -439,6 +442,16 @@ func parseNode2Port(appState *state.State) (m map[string]int, err error) {
 		}
 	}
 
+	// memberlist doesn't match raft join
+	if len(appState.ServerConfig.Config.Raft.Join) != len(appState.Cluster.AllNames()) {
```

Comment:
question: Should we instead do discovery if and only if `len(...Raft.Join) != 0` ? My reasoning is that if we have any parameters in `RAFT_JOIN` that means the user or the helm chart specified it and we should it. Otherwise if we have no parameter then let's do discovery. Is that assumption wrong ?

Reply:
i have been back and forth in this , if we should overwrite, but the default is going to be 1 node, what we can do i think check also if `appState.ServerConfig.Config.Raft.Join >= 1 ` before discovery, wdyt ?

Reply:
Yup I agree, i see you added the change already ;)